{"is_answered": false, "view_count": 34, "tags": ["python", "pandas", "unicode"], "last_activity_date": 1469719922, "answer_count": 0, "creation_date": 1469719922, "score": 0, "link": "http://stackoverflow.com/questions/38640800/unicodedecodeerror-error-when-displaying-dataframe-on-ipython-notebook", "owner": {"user_id": 6650391, "profile_image": "https://www.gravatar.com/avatar/78be80f1e81829416b25918b1380d8c3?s=128&d=identicon&r=PG", "user_type": "registered", "reputation": 11, "link": "http://stackoverflow.com/users/6650391/akotronis", "display_name": "akotronis"}, "title": "UnicodeDecodeError error when displaying dataframe on ipython notebook", "question_id": 38640800}